pkgs/tools/system/netdata/mosquitto.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, fetchFromGitHub, c-ares, cmake, libwebsockets_3_2, openssl }:
```

Comment:
Could you add some comment how does this differ from mosquitto 1.6.10 from nixpkgs and why it cannot be used?

Reply:
Do you mean add a comment here or in the actual file? 

In any case, the Netdata developers seem to have decided that it is necessary for them to fork Mosquitto. From what I can tell, this was done to accommodate some external callback functionality they require. Trying to build Netdata with the regular version of Mosquitto fails because of missing symbols/functions.

Reply:
I guess maybe a comment near the `preConfigure` linking the issue should be enough.

Reply:
I have added a comment to the top of the `mosquitto.nix` file as I think that makes the most sense. If you disagree please unresolve this conversation.